unic/ucd/ident/src/lib.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![forbid(unsafe_code, unconditional_recursion, missing_docs)]
+
+//! # UNIC — UCD — Identifier Properties
+//!
+//! A component of [`unic`: Unicode and Internationalization Crates for Rust](/unic/).
+//!
+//! Accessor to the UCD properties used widely by [UAX31 Unicode Identifier and Pattern Syntax].
+//!
+//! # Features
+//!
+//! - `xid` (default): the `XID_Start` and `XID_Continue` properties
+//! - `id` (optional): the `ID_Start` and `ID_Continue` properties;
+//!   _in most cases, you should prefer using the `XID` properties
+//!   because they are consistent under NFKC normalization_
+//! - `pattern` (optional): the `Pattern_Syntax` and `Pattern_White_Space` properties
+//!
+//! [UAX31 Unicode Identifier and Pattern Syntax]: <http://www.unicode.org/reports/tr31/>
+
+#[macro_use]
+extern crate unic_char_property;
+#[macro_use]
+extern crate unic_char_range;
+extern crate unic_ucd_core;
+
+
+#[cfg(feature = "xid")]
+mod xid {
```

Comment:
This looks good when the module is this short. But would be great to expand over time with some basic unit tests, specially for ASCII, which is very relevant here. Not a blocker, though.

Reply:
Since the tests would be for external API, it seems (to me) like they'd be better suited in the tests directory. But yeah, this can be addressed later.